src/System.Private.CoreLib/src/System/Threading/Tasks/AsyncCausalityTracer.cs

```diff
@@ -2,107 +2,69 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-//
-//
+#if FEATURE_COMINTEROP
```

Comment:
Change this to condition in the project file

Reply:
Done that already just forgot to remove the #if